t/feature_tracking.t

```diff
@@ -0,0 +1,283 @@
+END_PERL
+	  [
+		'PPI::Statement::Sub'   => 'sub meep(&$) {}',
+		'PPI::Token::Word'      => 'sub',
+		'PPI::Token::Word'      => 'meep',
+		'PPI::Token::Signature' => '(&$)',
+		'PPI::Structure::Block' => '{}',
+		'PPI::Token::Structure' => '{',
+		'PPI::Token::Structure' => '}',
+		'PPI::Statement::Sub'   => 'sub marp($left, $right) {}',
+		'PPI::Token::Word'      => 'sub',
+		'PPI::Token::Word'      => 'marp',
+		'PPI::Token::Signature' => '($left, $right)',
+		'PPI::Structure::Block' => '{}',
+		'PPI::Token::Structure' => '{',
+		'PPI::Token::Structure' => '}',
+	  ],
+	  "document-level default features";
+}
+
+DISABLE_FEATURE: {
+	test_document
+	  [ feature_mods => { signatures => 1 } ],
+	  <<'END_PERL',
+		sub meep(&$) {}
+		no feature 'signatures';
+		sub marp($left, $right) {}
+END_PERL
+	  [
+		'PPI::Statement::Sub'       => 'sub meep(&$) {}',
+		'PPI::Token::Word'          => 'sub',
+		'PPI::Token::Word'          => 'meep',
+		'PPI::Token::Signature'     => '(&$)',
+		'PPI::Structure::Block'     => '{}',
+		'PPI::Token::Structure'     => '{',
+		'PPI::Token::Structure'     => '}',
+		'PPI::Statement::Include'   => q|no feature 'signatures';|,
+		'PPI::Token::Word'          => 'no',
+		'PPI::Token::Word'          => 'feature',
+		'PPI::Token::Quote::Single' => q|'signatures'|,
+		'PPI::Token::Structure'     => ';',
+		'PPI::Statement::Sub'       => 'sub marp($left, $right) {}',
+		'PPI::Token::Word'          => 'sub',
+		'PPI::Token::Word'          => 'marp',
+		'PPI::Token::Prototype'     => '($left, $right)',
+		'PPI::Structure::Block'     => '{}',
+		'PPI::Token::Structure'     => '{',
+		'PPI::Token::Structure'     => '}',
+	  ],
+	  "disabling of features";
+}
+
+PROTOTYPE_ATTR: {
+	test_document
+	  <<'END_PERL',
+		sub meep :prototype(&$) {}
+END_PERL
+	  [
+		'PPI::Statement::Sub'   => 'sub meep :prototype(&$) {}',
+		'PPI::Token::Word'      => 'sub',
+		'PPI::Token::Word'      => 'meep',
+		'PPI::Token::Operator'  => ':',
+		'PPI::Token::Attribute' => 'prototype(&$)',
+		'PPI::Structure::Block' => '{}',
+		'PPI::Token::Structure' => '{',
+		'PPI::Token::Structure' => '}',
+	  ],
+	  "prototype attribute";
+}
+
+SYNTAX_KEYWORD_TRY: {
+	test_document
+	  <<'END_PERL',
+		use Syntax::Keyword::Try;
+		try{}catch{}
+END_PERL
+	  [
+		'PPI::Statement::Include'  => 'use Syntax::Keyword::Try;',
+		'PPI::Token::Word'         => 'use',
+		'PPI::Token::Word'         => 'Syntax::Keyword::Try',
+		'PPI::Token::Structure'    => ';',
+		'PPI::Statement::Compound' => 'try{}catch{}',
+		'PPI::Token::Word'         => 'try',
+		'PPI::Structure::Block'    => '{}',
+		'PPI::Token::Structure'    => '{',
+		'PPI::Token::Structure'    => '}',
+		'PPI::Token::Word'         => 'catch',
+		'PPI::Structure::Block'    => '{}',
+		'PPI::Token::Structure'    => '{',
+		'PPI::Token::Structure'    => '}',
+	  ],
+	  "Syntax::Keyword::Try";
+}
+
+CORE_TRY: {
+	test_document
+	  <<'END_PERL',
+		use feature "try";
+		try{}catch{}
```

Comment:
```perl
use 5.36.0;
use feature "try";
no warnings 'experimental::try';
try { } catch { }
```

> syntax error at try.pl line 4, near "catch {"
try.pl had compilation errors.

It compiles when the signature is included.

`try { } catch ($e) { }`

Reply:
Thanks, i've added it to the test and it seems to parse fine with the signature. :)